hphp/hack/test/typecheck/xhp_modifier/xhp_modifier_not_on_enum.php

```diff
@@ -0,0 +1,4 @@
+<?hh // strict
+
+# this is not allowed
```

Comment:
Could you make your comments more explicit? I can see it's not allowed by looking at the `.exp` file, but something like "the xhp keyword only applies to classes" would be more helpful here.